source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
If it's during `require` then things are though, and they're being a bit of a shit.
If it's only when you `new Mixpanel()` then it's easy `typeof document !== 'undfined' && new Mixpanel()` or likewise